src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyAloneOrSingleLineBlocksWithSemi.java

```diff
@@ -28,4 +28,18 @@ public TestClass2(String someValue) {
     public void testMethod11() {
     }
     ;
+
+    public @interface TestAnnnotation5 {
```

Comment:
Please update comment at top of Input to have new token.
One day we will automate reading of config from that comment so it will be easy to make Inputs, and you will not need to read UT at all, Inputs will be self descriptive

Reply:
Done